src/main/java/com/conveyal/analysis/GenerateRegionalAnalysisResults.java

```diff
@@ -0,0 +1,64 @@
+package com.conveyal.analysis;
+
+import com.conveyal.analysis.controllers.RegionalAnalysisController;
+import com.conveyal.analysis.models.OpportunityDataset;
+import com.conveyal.analysis.models.RegionalAnalysis;
+import com.conveyal.analysis.persistence.Persistence;
+import com.conveyal.file.FileStorage;
+import com.conveyal.file.FileStorageFormat;
+import com.conveyal.r5.analyst.progress.ProgressListener;
+import com.conveyal.r5.analyst.progress.TaskAction;
+import com.mongodb.DBObject;
+import com.mongodb.QueryBuilder;
+import org.mongojack.DBCursor;
+
+import java.util.Objects;
+
+public class GenerateRegionalAnalysisResults implements TaskAction {
+    private final FileStorage fileStorage;
+    public GenerateRegionalAnalysisResults (FileStorage fileStorage) {
+        this.fileStorage = fileStorage;
+    }
+
+    @Override
+    public void action(ProgressListener progressListener) throws Exception {
+        DBObject query = QueryBuilder.start().or(
+                QueryBuilder.start("travleTimePercentiles").is(null).get(),
+                QueryBuilder.start("cutoffsMinutes").is(null).get(),
+                QueryBuilder.start("destinationPointSetIds").is(null).get()
+        ).get();
+        try (DBCursor<RegionalAnalysis> cursor = Persistence.regionalAnalyses.find(query)) {
+            while (cursor.hasNext()) {
+                RegionalAnalysis regionalAnalysis = cursor.next();
```

Comment:
It might be good to add some logging in here so we can verify the effects of the migration when it runs. Or maybe better, enable DEBUG level logging when we run the migration to hit the log statements inside `getSingleCutoffGrid`.

Reply:
Added a few log statements in https://github.com/conveyal/r5/pull/957/commits/c2cc8f4a25dcc03ac27ed56201d9d85e611be7c1

Reply:
Thanks for adding some logging. To clarify, the underlying reason I was interested in logging was to observe whether the effects of the migration matched expectations. Rather than just counts, something that would tip us off if for example records were being missed due to an incorrect field name, or extra files were being produced etc. This is why I mentioned enabling debug for `getSingleCutoffGrid` as it would log each new file created and stored. Maybe we could also log the key fields of each regionalAnalysis record that was migrated.